mysql/lib/client.ts

```diff
@@ -212,7 +213,7 @@ export class AwsMySQLClient extends AwsClient {
       },
       null
     );
-    await this.releaseResources();
+    await PluginManager.releaseResources();
```

Comment:
`PluginManager.releaseResources()` is made static because it is meant to be called at the end of the application. It is not supposed be called every time we close a connection. Can you please verify if we are supposed to call `this.releaseResources` here.